### ISSUE TITLE
Crash on new assert from #5587

### ISSUE THREAD

**Issue by janisozaur** [OpenRCT2/OpenRCT2#5602]
**OS:** Windows
**Commit/Build:**  25b6c61

#5578 introduced [new assert](https://github.com/OpenRCT2/OpenRCT2/blob/25b6c61/src/openrct2/network/network.cpp#L1326-L1327):

```c
// If our tick is higher than the command tick we are in trouble.
assert(game_command_queue.begin()->tick >= gCurrentTicks);
```

which is getting hit in live builds.

Here's the dump, provided to me by one of the users. It's quite large, so I had to split it, sorry.

[openrct2.DMP.7z.aa.txt](https://github.com/OpenRCT2/OpenRCT2/files/1069361/openrct2.DMP.7z.aa.txt)
[openrct2.DMP.7z.ab.txt](https://github.com/OpenRCT2/OpenRCT2/files/1069365/openrct2.DMP.7z.ab.txt)
[openrct2.DMP.7z.ac.txt](https://github.com/OpenRCT2/OpenRCT2/files/1069360/openrct2.DMP.7z.ac.txt)
[openrct2.DMP.7z.ad.txt](https://github.com/OpenRCT2/OpenRCT2/files/1069362/openrct2.DMP.7z.ad.txt)
[openrct2.DMP.7z.ae.txt](https://github.com/OpenRCT2/OpenRCT2/files/1069363/openrct2.DMP.7z.ae.txt)
[openrct2.DMP.7z.af.txt](https://github.com/OpenRCT2/OpenRCT2/files/1069364/openrct2.DMP.7z.af.txt)
[openrct2.DMP.7z.ag.txt](https://github.com/OpenRCT2/OpenRCT2/files/1069366/openrct2.DMP.7z.ag.txt)
[openrct2.DMP.7z.ah.txt](https://github.com/OpenRCT2/OpenRCT2/files/1069367/openrct2.DMP.7z.ah.txt)

Here are sha1sums so you can verify integrity of the archives:
```
503bec306ea13ad0400cf84a863c80c73625bd28  openrct2.DMP
72ca43f8149ac304a4caddfa838ae8be5b267d9d  openrct2.DMP.7z
d20f1548c2d38b4839999267ddebebab50c10db7  openrct2.DMP.7z.aa.txt
3026bae4a58c64e5d430157c075783c836a55b84  openrct2.DMP.7z.ab.txt
86cf3b2b2753231902cdf6089ecd0fda66f1bcef  openrct2.DMP.7z.ac.txt
4f6038ead4443602dd68eb65f167340357c2b1b7  openrct2.DMP.7z.ad.txt
af4aa2523378d18d8b027b66e1f98138d338f92c  openrct2.DMP.7z.ae.txt
fd3a435b9ca593c350655a026eae9f34da4c309a  openrct2.DMP.7z.af.txt
32e65924b682216913c908c96292d23a658465af  openrct2.DMP.7z.ag.txt
b28b5df6efcad70736b63f1ab8dd1d21a39442be  openrct2.DMP.7z.ah.txt
```

**Comment by janisozaur** [comment]
Ping @ZehM4tt as author of #5578

**Comment by janisozaur** [comment]
[openrct2-symbols-release-win32-25b6c61.zip](https://github.com/OpenRCT2/OpenRCT2/files/1069376/openrct2-symbols-release-win32-25b6c61.zip)

I was told this happening with 32 bit build, here are the symbols

**Comment by jensj12** [comment]
Isn't this simply because of slow internet connections, causing the command to arrive after the execution tick has passed? If so, it is not possible to avoid this assert completely.
I'd rather see a warning being logged instead of crashing the game.

**Comment by ZehM4tt** [comment]
The assert means that the command in the queue will never be executed as you already ran past the tick number. There is no way to actually fix this, it would be more interesting to on how you are getting there.

**Comment by janisozaur** [comment]
Fixed in #5631
